tests/observability/metrics/conftest.py

```diff
@@ -593,3 +596,39 @@ def vm_created_pod_total_initial_metric_value(prometheus, namespace):
             prometheus=prometheus, metrics_name=KUBEVIRT_VM_CREATED_BY_POD_TOTAL.format(namespace=namespace.name)
         )
     )
+
+
+@pytest.fixture()
+def non_evictable_data_volume(namespace, unprivileged_client):
```

Comment:
thank you for this pr.
"non_evictable" is releavnt to a vm - please replace with a more meaningful name tha also explains which os is used, e.g: `fedora_rwo_dv` or alike